packages/eas-cli/src/commands/update/index.ts

```diff
@@ -587,7 +589,7 @@ export default class UpdatePublish extends EasCommand {
               ? [{ label: 'Android update ID', value: newAndroidUpdate.id }]
               : []),
             ...(newIosUpdate ? [{ label: 'iOS update ID', value: newIosUpdate.id }] : []),
-            { label: 'Message', value: message! },
+            { label: 'Message', value: truncatedMessage! },
```

Comment:
Is the `!` still necessary, as `truncatePublishUpdateMessage` always returns a string according to the return type?

Reply:
moving the bang further up. I assigned the `undefined` message to an empty string, which I think is a less elegant way of mimicking the `!` operator on typescript's type checking.